src/index.tsx

```diff
@@ -93,10 +105,17 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
       pollingIntervalMs: config.pollingIntervalMs ?? undefined,
     };
 
-    EppoReactNativeClient.instance.setLogger(config.assignmentLogger);
+    EppoReactNativeClient.instance.setAssignmentLogger(config.assignmentLogger);
+
+    const storageKeySuffix = buildStorageKeySuffix(config.apiKey);
+    const assignmentCache = assignmentCacheFactory({
+      storageKeySuffix,
+    });
+    if (assignmentCache instanceof HybridAssignmentCache) {
```

Comment:
`assignmentCacheFactory` return type is `AssignmentCache`, which doesn't have an `init` method, so checking that it's an instance of `HybridAssignmentCache` lets us invoke that method in typescript

Reply:
This is fine for now as it parallels what we've got in js-client. When we bake these into common, we can rethink the interface a bit (ex: `interface InitializableCache { function init(): void {}}` or `if ('init' in assignmentCache) {...}`